app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/SkillListingPresenter.kt

```diff
@@ -79,6 +86,53 @@ class SkillListingPresenter : ISkillListingPresenter,
         skillListingView?.displayError()
     }
 
+    override fun onSkillMetricsFetchSuccess(response: Response<ListSkillMetricsResponse>) {
+        skillListingView?.visibilityProgressBar(false)
+        if (response.isSuccessful && response.body() != null) {
+            Timber.d("METRICS FETCHED")
+            metrics = response.body().metrics
+            if (metrics != null) {
+                metricsList.clear()
+                if (metrics?.rating != null) {
+                    if (metrics?.rating?.size as Int > 0) {
+                        metricsList.add(Pair("SUSI, What are your highest rated skills?", metrics?.rating))
```

Comment:
@arundhati24 here you can use utilmodel to get the string

Reply:
Okay @batbrain7 I will try doing this. Thanks :)